app/com/lynxanalytics/biggraph/graph_operations/ExecuteSQL.scala

```diff
@@ -46,16 +43,34 @@ object ExecuteSQL extends OpFromJson {
     catalog
   }
 
+  private def createTempViews(protoTables: Map[String, ProtoTable]): Unit = {
```

Comment:
This scary change is due to https://github.com/apache/spark/commit/7953fcdb56abd2e574ea7fdfed925fe06dffdec0. ([SPARK-34700](https://issues.apache.org/jira/browse/SPARK-34700)) The LogicalPlan cannot be directly passed to `catalog.createTempView()`. The idea of this workaround is from that commit.